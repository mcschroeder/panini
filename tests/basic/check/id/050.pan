id : (i:ℤ) → { j:ℤ | j = i }

f50 : {v:int|false}
f50 = id 0

