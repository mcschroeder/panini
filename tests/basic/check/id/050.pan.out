tests/basic/check/id/050.pan:4:1-4: error: unable to verify type of f50